atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransactionManager.java

```diff
@@ -284,6 +286,7 @@ public static TransactionManager create(
             Supplier<AtlasDbConstraintCheckingMode> constraintModeSupplier,
             ConflictDetectionManager conflictDetectionManager,
             SweepStrategyManager sweepStrategyManager,
+            TableMetadataManager tableMetadataManager,
```

Comment:
Breaks internal large product - think it's fine as sometimes they might not want to warm the cache.